lib/modules/idea-single-widgets/lib/vote-overview.js

```diff
@@ -1,18 +1,28 @@
 const { parse } = require('json2csv');
 const rp = require('request-promise');
-
+const moment = require('moment');
 module.exports = {
     generateCsv: (votes) => {
-        const fields = ['id', 'ideaId', 'userId', 'confirmed', 'opinion', 'ip', 'createdAt'];
-        const options = { fields };
+        const fields = ['Vote ID', 'User ID', 'Mening', 'Postcode', 'IP', 'Datum', 'OK'];
+
+        const mappedVotes = votes.map((vote) => {
+            return {
+                'Vote ID': vote.id,
+                'User ID': vote.userId,
+                'Mening': vote.opinion,
```

Comment:
Kan hier nog een check bij om deze 'yes' of 'no' naar de nederlandse 'voor' en 'tegen' om te zetten? Dan klopt 'ie met hoe het op de site staat